### ISSUE TITLE
RequireClaim method signature is a little confusing

### ISSUE THREAD

**Issue by benmccallum** [aspnet/Security#1883]
```csharp
public AuthorizationPolicyBuilder RequireClaim(string claimType, params string[] requiredValues);
public AuthorizationPolicyBuilder RequireClaim(string claimType, IEnumerable<string> requiredValues);
```

"requiredValues" is a confusing name as the method in reality only checks for the presence of one of these values, not all. The parameter description is perfect, but I'm not sure if that's enough. I certainly got confused until I confirmed it in the docs/saw comment. 

It's a petty thing I know, but maybe something more generic like "values" would suffice? Or go super explicit like "anyOfTheseValues"?

**Comment by HaoK** [comment]
Sure, do you want to submit a PR for this change?

**Comment by benmccallum** [comment]
Sure thing. Any preference on a new param name?

**Comment by dgarciarubio** [comment]
How about "possibleValues"? just "values" seems vague, but "anyOfTheseValues" sounds too informal to me.

**Comment by HaoK** [comment]
acceptedValues maybe?

**Comment by benmccallum** [comment]
How about valuesAllowed? Or along the lines of auth, valuesAuthorized?

**Comment by HaoK** [comment]
allowedValues is fine

**Comment by benmccallum** [comment]
PR up with allowedValues. Cheers!

**Comment by HaoK** [comment]
Merged into master for 3.0, thanks @benmccallum !
